index.bs

```diff
@@ -837,11 +837,26 @@ When an {{XRReferenceSpace}} is requested, the user agent MUST <dfn>create a ref
 
   1. Let |session| be the {{XRSession}} object that requested creation of a reference space.
   1. Let |type| be set to the {{XRReferenceSpaceType}} passed to {{requestReferenceSpace()}}.
-  1. If |type| is {{bounded-floor}}, let |referenceSpace| be a new {{XRBoundedReferenceSpace}}.
-  1. Else let |referenceSpace| be a new {{XRReferenceSpace}}.
-  1. Initialize |referenceSpace|'s [=XRReferenceSpace/type=] to be |type|.
-  1. Initialize [=XRSpace/session=] to be |session|.
-  1. Return |referenceSpace|.
+  1. If the [=reference space is supported=] for |type| and |session|, run the following steps:
+    1. If |type| is {{bounded-floor}}, let |referenceSpace| be a new {{XRBoundedReferenceSpace}}.
+    1. Else let |referenceSpace| be a new {{XRReferenceSpace}}.
+    1. Initialize |referenceSpace|'s [=XRReferenceSpace/type=] to be |type|.
+    1. Initialize [=XRSpace/session=] to be |session|.
+    1. Return |referenceSpace|.
+  1. Else, return <code>null</code>.
+
+</div>
+
+<div class="algorithm" data-algorithm="reference-space-supported">
+To check if a <dfn>reference space is supported</dfn> for a given reference space type |type| and {{XRSession}} |session|, run the following steps:
+
+
+  1. If |type| is {{inline}}, return <code>true</code>.
+  1. If |type| is {{local}} or {{local-floor}}, and |session| is an [=immersive session=], return <code>true</code>.
+  1. If |type| is {{local}} or {{local-floor}}, and the [=/XR device=] supports reporting orientation data, return <code>true</code>.
+  1. If |type| is {{bounded-floor}} and |session| is an [=immersive session=], return if [=bounded reference spaces are supported=] by the [=/XR device=].
+  1. If |type| is {{unbounded}}, |session| is an [=immersive session=], and the [=/XR device=] supports reporting position and orientation data, return <code>true</code>.
```

Comment:
I'm don't think this is an adequate requirement for `unbounded`, but I'm not sure how to describe it better. @thetuvix, do you have any thoughts about an appropriate support test here?

Reply:
For the sake of moving this PR forward, I propose we change this to something along the lines of "supports stable tracking near the user over an unlimited distance, return <code>true</code>."

If there's a more accurate way to describe the requirements for an `unbounded` space then we can fix it up later, but that should be good enough for now and prevent implementers from saying "Oh, by this definition my Vive *totally* supports `unbounded`!"

Reply:
Fixed.